docs/api/example-query.md

```diff
@@ -34,30 +35,76 @@ The server returns the following JSON:
       "edges": [
         {
           "node": {
-            "name": "Ford Inc",
-            "price": {
-              "amount": 64.98
+            "id": "UHJvZHVjdDo3Mg==",
+            "name": "Apple Juice",
+            "description": "Fell straight from the tree, on to Newton’s head, then into the bottle. The autumn taste of English apples. Brought to you by gravity.",
+            "category": {
+              "name": "Groceries"
             }
           }
         },
         {
           "node": {
-            "name": "Rodriguez Ltd",
-            "price": {
-              "amount": 18.4
+            "id": "UHJvZHVjdDo3NA==",
+            "name": "Banana Juice",
+            "description": "Build your protein the natural way, with exotic banana juice made from ripe fruit and packed with all the goodness of the tropical sun.",
+            "category": {
+              "name": "Groceries"
             }
           }
         },
         {
           "node": {
-            "name": "Smith Inc",
-            "price": {
-              "amount": 48.66
+            "id": "UHJvZHVjdDo3OQ==",
+            "name": "Bean Juice",
+            "description": "Bean there, drunk that! The energy drink for the health-conscious. Brand new bean juice; from allotment to bottle in under 8 hours.",
+            "category": {
+              "name": "Groceries"
             }
           }
         }
       ]
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Example mutation
+
+Let's look at an example of a mutation. Imagine that we want to update the name of a product. In Saleor API this can be done with the `productUpdate` mutation:
+
+```graphql
+mutation {
+  productUpdate(
+    id: "UHJvZHVjdDo3Mg=="
+    input: { name: "Apple Juice - Summer Edition" }
+  ) {
+    errors {
```

Comment:
Shouldn't we keep the "convention" of putting the errors at the end of the query?